src/common/translation/i18n/en.json

```
@@ -480,15 +480,9 @@
       }
     },
     "preliminaryForm": {
-      "childBirthdate": {
+      "childBirthyear": {
         "input": {
-          "day": {
-            "placeholder": "date"
-          },
-          "label": "Child's date of birth (dd.mm.yyyy)",
-          "month": {
-            "placeholder": "month"
-          },
+          "label": "Child's year of birth",
           "year": {
             "placeholder": "year"
```

```suggestion
            "placeholder": "Enter child's year of birth"
```
to make the placeholder more similar to other placeholders in the UI